config/initializers/omniauth.rb

```diff
@@ -1,4 +1,6 @@
+#this file is used to store the variable needed for omniauth login and signup
 Rails.application.config.middleware.use OmniAuth::Builder do
-  provider :google_oauth2, ENV["google_app_key"],ENV["google_app_secret"] , skip_jwt: true
-  provider :facebook, ENV["APP_ID"],ENV["APP_SECRET"] , skip_jwt: true
+  #the provider is google_oauth2 and the app-key of google developers app is stored in OAUTH_GOOGLE_APP_KEY
+  #the app-secret of google developers app is stored in variable OAUTH_GOOGLE_APP_SECRET
+  provider :google_oauth2, ENV["OAUTH_GOOGLE_APP_KEY"],ENV["OAUTH_GOOGLE_APP_SECRET"] , skip_jwt: true
```

Comment:
Ah ok i see it here. Great.

Reply:
Just take care of Google today. We will figure out facebook in later prs.

Reply:
I think possibly the changes here to `OAUTH_GOOGLE_APP_KEY` make the `application.yml` unnecessary, as this should fetch the environment variable. See for example: https://github.com/publiclab/plots2/blob/master/config/initializers/recaptcha.rb